packages/yoastseo/src/languageProcessing/languages/de/helpers/checkIfWordIsComplex.js

```diff
@@ -1,18 +1,17 @@
-import wordComplexity from "../config/internal/wordComplexity";
-import functionWords from "../config/functionWords";
-
 const suffixes = "(en|e|s)$";
 const suffixesRegex = new RegExp( suffixes );
 
 /**
  * Checks if a word is complex.
+ * This is a helper for the premium assessment Word Complexity. As such, this helper is registered from the premium repository.
  *
+ * @param {object} wordComplexityConfig The config needed for assessing the word complexity.
  * @param {string} word The word to check.
+ * @param {object} functionWords The function words list.
  *
  * @returns {boolean} Whether or not a word is complex.
  */
-export default function checkIfWordIsComplex( word ) {
-	const wordComplexityConfig = wordComplexity;
+export default function checkIfWordIsComplex( wordComplexityConfig, word, functionWords ) {
```

Comment:
Accessing the `wordComplexityConfig` from the researcher means:
* It allows us to eventually register the config to the researcher from premium/shopify.
* This way, the config is not bundled in Free.

Accessing `functionWords` from the researcher instead of directly importing it means:
* When we eventually register `checkIfWordIsComplex` helper from premium/shopify, the function words list is not going to be bundled - reducing size.

Reply:
That's a good point! Yes, I think adding `functionWords` together with `wordComplexityConfig` would work. 👍🏽